schemas/blocks/interstitial.json

```diff
@@ -10,6 +10,9 @@
       "number": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
+      "page_title": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+      },
```

Comment:
Any reason this is not added to all block types?.

Reply:
No - and it probably should be

Reply:
Done